lib/engine/game/g_1862/step/track.rb

```diff
@@ -25,9 +25,9 @@ def lay_tile_action(action, entity: nil, spender: nil)
             super
           end
 
-          def upgraded_track(_from, to, _hex)
+          def track_upgrade?(_from, to, _hex)
```

Comment:
why are you changing other games?

Reply:
This is because I changed `upgraded_track` to remove the side effect in the base `Tracker`, since 1862 and 18CZ override `upgraded_track` I've updated them accordingly